packages/browser/core/src/core/auth/internal/StitchAuthImpl.ts

```diff
@@ -140,12 +265,108 @@ export default class StitchAuthImpl extends CoreStitchAuth<StitchUser>
       });
     }
   }
+
+  public setRedirectFragment(key: RedirectKeys | RedirectFragmentFields, value: any) {
```

Comment:
[nit] don't make this public
[nit] I don't quite get what this function name means or does. I'm not sure why RedirectFragmentFields would ever be passed in here for storage.

Reply:
I think a better name is `setRedirectAuthValue`. This also should be taking its own type. It's not a redirect key because it's not for the window URL (that can change in the future and would break clients). It's also not a RedirectFragmentFields for the same reason. Maybe `RedirectAuthRequestFields`

Reply:
Another thought. It doesn't look this has much use if it just calls storageRef.set. If this were to apply some prefix prior to storage, then this could server a use. it would need a get equivalent to fetch with prefix.

Reply:
👍 removed